spec/controllers/items_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
```

Comment:
__[rubocop]__
* Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

<!-- Comment by [SideCI](https://sideci.com) -->